hotshot-query-service/src/data_source/update.rs

```diff
@@ -129,19 +130,26 @@ where
                 }
 
                 let (vid_common, vid_share) = match vid_share {
-                    Some(VidDisperseShare::V0(share)) => (
+                    Some(VidDisperseShare::V1(share)) => (
```

Comment:
Is there any reason why we don't want to keep V0/V1/V2??

Reply:
just nasty version alignment tech debts, [1059546](https://github.com/EspressoSystems/espresso-network/pull/3721/commits/105954699a0bc3a82fa28dc4a4bca1dde21ee6a9)